kratos/utilities/variable_utils.cpp

```diff
@@ -486,4 +487,94 @@ void VariableUtils::AuxiliaryAtomicAdd(
         rSumValue[2] += rPrivateValue[2];
 }
 
+template <>
+ModelPart::ElementsContainerType& VariableUtils::GetContainer<ModelPart::ElementsContainerType>(ModelPart& rModelPart)
+{
+    return rModelPart.Elements();
+}
+
+template <>
+ModelPart::ConditionsContainerType& VariableUtils::GetContainer<ModelPart::ConditionsContainerType>(ModelPart& rModelPart)
+{
+    return rModelPart.Conditions();
+}
+
+template <class TDataType, class TContainerType, class TWeightDataType>
+void VariableUtils::DistributeVariable(
+    ModelPart& rModelPart,
```

Comment:
I think it would be easier to directly pass the `TContainerType`. Then an auxiliary method or a lambda can be used in the Python export to be compatible with both elements and conditions.

Reply:
Only problem is, If I am to use container as an argument, I again need the model part as well. Because,

First I need to clear the output variable of nodes to have zeros.
Secondly, I need the MPI communicator.

Thats why I created the GetContainer method, so everything can be obtained from model part.

Reply:
You are completely right. Forget about this comment (also about the one below).